src/tests/Microsoft.PowerFx.Core.Tests.Shared/UserDefinedTypeTests.cs

```diff
@@ -126,6 +135,23 @@ public void TestUDTErrors(string typeDefinition, int expectedErrorCount, string
             Assert.All(errors, e => Assert.Contains(expectedMessageKey, e.MessageKey));
         }
 
+        [Theory]
+
+        // not in Type literal expression;
+        [InlineData("Points := Type([{ x: Number, y: Number }]); F(): Points = RecordOf(Points); ", "ErrknownTypeHelperFunction")]
+        [InlineData("Points := Type([{ x: Number, y: Number }]); F(): Number = RecordOf(Points).x; ", "ErrknownTypeHelperFunction")]
+
+        // RecordOf record type
+        [InlineData("Point := Type({ x: Number, y: Number }); PointR := Type(RecordOf(Point)); ", "ErrNamedType_InvalidTypeDefinition")]
```

Comment:
>Type(RecordOf(Point)) [](http://example.com/codeflow?start=72&length=21)

Can you have inline type declaration within RecordOf?

For example,
Type(RecordOf(Type([{ x: Number, y: Number }]))

Reply:
no